tests/chocolatey-tests/commands/choco-list.Tests.ps1

```diff
@@ -247,253 +80,39 @@ Describe "choco <_>" -ForEach $Command -Tag Chocolatey, ListCommand, SearchComma
-        }
-
-        It "Should contain a summary" {
-            $Output.Lines | Should -Contain "0 packages found."
-        }
-    }
-
-    # Issue: https://gitlab.com/chocolatey/collaborators/choco-licensed/-/issues/530 (NOTE: Proxy bypassing also works on Chocolatey FOSS)
-    Context "Listing packages on source using proxy and proxy bypass list" -Skip:(!$licensedProxyFixed) {
+    Context "Listing local packages with paging" {
         BeforeAll {
             Restore-ChocolateyInstallSnapshot
-            $null = Invoke-Choco config set --name=proxy --value="https://invalid.chocolatey.org/"
-            $null = Invoke-Choco config set --name=proxyBypassList --value="hermes.chocolatey.org"
-
-            $Output = Invoke-Choco $_ mvc
-        }
-
-        It "Exits with Success (0)" {
-            $Output.ExitCode | Should -Be 0
-        }
-
-        It "Displays the package <_>" -ForEach @("mvcmusicstore-db 1.2.0"; "mvcmusicstore-web 1.2.0") {
-            $Output.Lines | Should -Contain $_
-        }
-
-        It "Displays amount of packages found" {
-            $Output.Lines | Should -Contain "2 packages found."
-        }
-    }
 
-    # Issue: https://gitlab.com/chocolatey/collaborators/choco-licensed/-/issues/530 (NOTE: Proxy bypassing also works on Chocolatey FOSS)
-    Context "Listing packages on source using proxy and proxy bypass list on command" -Skip:(!$licensedProxyFixed) {
-        BeforeAll {
-            Restore-ChocolateyInstallSnapshot
-            $null = Invoke-Choco config set --name=proxy --value="https://invalid.chocolatey.org/"
+            $null = Invoke-Choco install hasdependency --confirm
 
-            $Output = Invoke-Choco $_ mvc "--proxy-bypass-list=hermes.chocolatey.org"
+            $Output = Invoke-Choco list dependency --page 1 --page-size 2 --id-only
         }
 
         It "Exits with Success (0)" {
             $Output.ExitCode | Should -Be 0
         }
 
-        It "Displays the package <_>" -ForEach @("mvcmusicstore-db 1.2.0"; "mvcmusicstore-web 1.2.0") {
-            $Output.Lines | Should -Contain $_
+        It "Should contain package isexactversiondependency" {
+            $Output.Lines | Should -Contain isexactversiondependency
         }
 
-        It "Displays amount of packages found" {
-            $Output.Lines | Should -Contain "2 packages found."
+        It "Should not contain package <_>" -Foreach @('isdependency', 'hasdependency') {
+            $Output.Lines | Should -Not -Contain $_
         }
     }
 
-    # Issue: https://github.com/chocolatey/choco/issues/2304
-    Context "Listing packages with exact and all version displayed without pre-release argument" -Skip:(-Not (Test-ChocolateyVersionEqualOrHigherThan "0.10.16-beta-233")) {
+    Context "Listing local packages with unsupported argument outputs warning" -ForEach @('-l', '-lo', '--lo', '--local', '--localonly', '--local-only', '--order-by-popularity', '-a', '--all', '--allversions', '--all-versions') {

```

Comment:
Should we maybe add a test to ensure `--limit-output` doesn't output these warnings?